lib/livebook_web/controllers/error_html.ex

```diff
@@ -3,13 +3,13 @@ defmodule LivebookWeb.ErrorHTML do
 
   def render("404.html", assigns) do
     ~H"""
-    <.error_page status={@status} title="No Numbats here!" />
+    <.error_page status="404" title="No Numbats here!" />
```

Comment:
```suggestion
    <.error_page status={404} title="No Numbats here!" />
```

So we always pass integer :)